PathManager/Src/PM_CommsWithAM.cpp

```diff
@@ -0,0 +1,69 @@
+/**
+ * Defines functions for sending data from the path manager to the attitude manager.
+ * Has function definitions used by both the attitude and path manager.
+ * Author: Anthony Bertnyk
+ */
+
+//#include "CommWithAttitudeManager.hpp"
+//#include "CommWithPathManager.hpp"
+#include "PM_CommsWithAM.hpp"
+
+
+extern "C"
+{
+#include "cmsis_os.h"
+}
+
+//Set up a mail queue for sending commands to the attitude manager
+osMailQDef(commandsMailQ, PATH_ATTITUDE_MAIL_Q_SIZE, CommandsForAM);
+osMailQId commandsMailQID;
```

Comment:
Have you thought about how AM receives this queue? If so I would like to replicate for SM queues

Reply:
Might be outdated as a lot of discussions have happened in the last ~10 days, but it makes sense for SM to create all mailQs and pass the relevant IDs to each manager.